int32 Quality
float32 sensor_values
float32 field_of_view
float32 min_range
float32 max_range
float32 range

